workers/unity/Packages/io.improbable.worker.sdk/CIO.cs

```diff
@@ -0,0 +1,178 @@
+using System.Runtime.InteropServices;
+using Improbable.Worker.CInterop.Internal;
+using Int64 = System.Int64;
+using Uint64 = System.UInt64;
+using Uint32 = System.UInt32;
+using Uint8 = System.Byte;
+using Char = System.Char;
+using IntPtr = System.IntPtr;
+
+namespace Improbable.Gdk
```

Comment:
I'd possible use the same namespace as the worker SDK to minimize changes required in the future. (`Improbable.Worker.CInterop.Internal` I think)

Reply:
Same thing applies for `CEventTrace.cs`